tools/chrome-debug-extension/README.md

```diff
@@ -0,0 +1,75 @@
+
+## Installing the tool
+
+1. Download either the [official](https://js.monitor.azure.com/release/tools/ai.chrome-ext.zip) or [nightly](https://js.monitor.azure.com/nightly/tools/ai.chrome-ext.zip) build and unzip it into a folder on your PR
+1. Open the Manage Extensions page in either Chrome or Edge
+   - Browse to either edge://extensions or chrome://extensions/ respectively
+   - Or choose Extensions from the ... menu
+1. Enable Developer Mode
+1. Click `Load unpacked`
+1. Browse to the path where you unzipped the extension
+1. Click `Select Folder`
+
+## Using the tool
+
+1. To launch the tool, click the ![Telemetry-Viewer-icon](images/icon-19.png) icon next to the address bar
+    - If it isn't there, click the `small puzzle piece` icon and then the `Telemetry Viewer`![Telemetry-Viewer-icon](images/icon-19.png)
+1. The first time you run the tool, it will start in the `Configuration Selection` page, which you can always return to by clicking the gear icon later
+    - If your team has a preset configuration in the drop down list, select it
+    - Otherwise, use you can start with the Default configuration
+    - You can also chooose to craft your own custom configuration, using one of the presets as a starting point by clicking `Copy to Custom Configuration`
+1. Click `OK` and you will be taken to the main page where you can view live events as they are generated
+
+## Exploring the tool
+1. Tooltips explain what each feature across the top does
+1. Clicking on an event will show details of that event in the bottom half of the tool
+1. You can save a session to a .json file in your Downloads folder, and open it again by dragging it back into the tool's window
+    - This is handy for attaching a session to a bug report for an engineer to be able to examine later
+
+## Updating the tool
+1. To get the latest version of the tool, follow step 1 of the installation steps again, and unzip it to the same folder that you orginally unzipped it to
+1. In the browser, go to the `Manage Extensions` page
+     - Browse to either edge://extensions or chrome://extensions/ respectively
+1. Click `Reload` under the `Telemetry Viewer` extension
+
+## Configuring the tool
+
+Get the most out of the tool by customizing its configuration for your project. You can read about all of the options in the comments in [this file](https://github.com/microsoft/ApplicationInsights-JS/blob/master/tools/chrome-debug-extension/src/configuration/IConfiguration.ts)
+
+You have several options for configurations:
+1. Use a preset configuration
+   - Ideal if your team already has one built that you can reuse - can be highly customized to your telemetry schema
+1. Use the default configuration
+   - Most suitable for situations where you are debugging the integration of Application Insights into your web application
+1. Use a local custom configuration
```

Comment:
Missed the numbering order here

Reply:
If you're referring to each line starting with 1. that is intentional, because MD viewers will show those as a numbered list, and it has the nice property that if you need to insert a new item in the middle, you don't have to manually renumber all of the following items.